integrations/access/servicenow/types.go

```diff
@@ -38,6 +52,11 @@ type Incident struct {
 	WorkNotes string `json:"work_notes"`
 }
 
+const (
+	ResolutionStateResolved = "6"
```

Comment:
nit: godoc
Also, please identify where these codes came from. Link to the API docs or any other references